manifest.json

```diff
@@ -9,6 +9,7 @@
     "scripting"
   ],
   "background": {
+    "scripts": ["background.js"],
     "service_worker": "background.js"
```

Comment:
This allows Firefox (and potentially Safari) to load the extension's background script in an event page while still loading the background script as a service worker in Chrome. Firefox will warn that the `"service_worker"` key is not supported and Chrome will warn that `"scripts"` is not supported, but both will load and run as expected.

Reply:
nice! I will test this later today and merge if it works